### DISCUSSION TITLE
Making a permenant way to stop the "Trial has ended" message

### DISCUSSION THREAD

**Discussion by SethFalco** [Kong/insomnia#4696] [Ideas] [answered]
I've been using Insomnia for a while, but have never really needed the paid features of it.

I was considering getting the paid plan literally just so the "Trial has ended" message would go away when I launch it sometimes. ^-^' However, I'm not sure if I'm willing to pay $50 a year for that.

Is there anything that can be done to get rid of that? ^-^'

---

I originally asked this in support, and was asked to open a discussion here to get other thoughts.

**Comment by subnetmarco** [comment, marked as answer]
This doesn't not apply anymore.
